tests/llm/utils/mock_utils.py

```diff
@@ -165,9 +190,15 @@ def load_test_cases(self) -> List[HolmesTestCase]:
                     config_dict["user_prompt"] = (
                         config_dict["user_prompt"] + extra_prompt
                     )
-                    test_case = TypeAdapter(AskHolmesTestCase).validate_python(
-                        config_dict
-                    )
+                    try:
```

Comment:
Consider moving it to the general try except block since it is reuse in all the cases